### DISCUSSION TITLE
What is the OpenAPI / swagger version of fastapi 0.63.0?

### DISCUSSION THREAD

**Discussion by L1uNan** [fastapi/fastapi#8801] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
What is the OpenAPI / swagger version of fastapi 0.63.0?
```

### Description

What is the OpenAPI / swagger version of fastapi 0.63.0?

### Operating System

macOS

### Operating System Details

What is the OpenAPI / swagger version of fastapi 0.63.0?

### FastAPI Version

0.63.0

### Python Version

3.9.2

### Additional Context

What is the OpenAPI / swagger version of fastapi 0.63.0?

**Comment by jiyolla** [comment]
I think you can install the 0.63.0 version(possibly in a venv)

and run a simplest app

open the docs page

check the 'openapi.json' which is available at the top left of docs page
![image](https://user-images.githubusercontent.com/71974321/130627711-cbde4e84-2355-40de-99b4-076b9c4cdc8e.png)

it has a version field.


This definitively is not the easiest way, but I think it will work out.

Following these steps, I had "openapi":"3.0.2" with fastapi=0.63.0

But maybe openapi version really is not only affected by FastAPI version.

**Comment by YuriiMotov** [comment, marked as answer]
The version of OpenaAPI is shown at the top of Swagger Docs page as @jiyolla has pointed out.

To get the version of Swagger UI, you need to view the source sode of Swagger Docs page

![Screenshot from 2025-06-25 12-31-46](https://github.com/user-attachments/assets/fb89cafa-74ab-42dc-92ba-513ba8e2d0fe)

then follow the URL that looks similar to `https://cdn.jsdelivr.net/npm/swagger-ui-dist@5/swagger-ui-bundle.js`

![Screenshot from 2025-06-25 12-32-12](https://github.com/user-attachments/assets/1f9782ff-9912-4b09-bc16-148dbd7d3873)

and search by `this.version="`

![Screenshot from 2025-06-25 12-32-31](https://github.com/user-attachments/assets/199deac2-bc0a-4147-b0b3-79de323ea292)

FastAPI doesn't usually pin the version of Swagger (in some versions it does pin it because of issues of Swagger UI). So, usually the latest version will be used
